src/React/DOM/Props.purs

```diff
@@ -297,6 +301,19 @@ radioGroup = unsafeMkProps "radioGroup"
 readOnly :: Boolean -> Props
 readOnly = unsafeMkProps "readOnly"
 
+ref :: String -> Props
+ref = unsafeMkProps "ref"
+
+-- | You can use `writeRef` to store a reference on `Refs`.
+-- | ``` purescrript
+-- | div [ refCb (writeRef this "inputEl") ] [...]
+-- | ```
+refCb
+  :: forall access eff
+   . (Node -> Eff (refs :: ReactRefs (write :: Write | access) | eff) Unit)
```

Comment:
Just a quick note, would it make sense for this to be `Nullable Node`?
https://facebook.github.io/react/docs/refs-and-the-dom.html#caveats